check_dependencies_v2.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python2
+
+import sys
+import imp
+
+modules = ['unittest',
+           'subprocess',
+           'resource',
```

Comment:
Is it possible to analyse imports in python files automatically to check all required imports in runtime? Just to avoid manual updates of the list